spec/unit/shoulda/matchers/active_record/have_db_column_matcher_spec.rb

```diff
@@ -98,6 +98,14 @@
     end
   end
 
+  context 'with invalid argument option' do
+    it 'throws an ArgumentError' do
+      expect {
+        have_db_column(:salary).with_options(preccision: 5)
+      }.to raise_error(ArgumentError)
```

Comment:
What do you think about testing what the error message is going to be rather than what kind of error it is? When you do that you might see that error could be improved a bit. I believe that as `invalid_options` is an array, when it gets interpolated it will appear like an array in the message. Not sure if that's what you were going after or not.

Reply:
Ah! Good observation!! I just changed the test and improved the message raised. Let me know what did you think about  it?